05_stringer/jimbotech/stringer_test.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+)
+
+type controlTest struct {
+	input    IPAddr
+	expected string
+}
+
+var hosts = []controlTest{
+	{IPAddr{127, 0, 0, 1}, "127.0.0.1"},
+	{IPAddr{8, 8, 8, 8}, "8.8.8.8"},
+	{IPAddr{}, "0.0.0.0"},
```

Comment:
Optional- You can add some more test such as` [4]byte{10}` should result in `10.0.0.0` and `[4]byte{255,255,255,255}` should result in `255.255.255.255`